### ISSUE TITLE
[Bug] "File not found" error is not being logged.

### ISSUE THREAD

**Issue by mayurikini** [altugbakan/rs-tftpd#31]
The "File not found" error isn't appearing in the log as expected.

I have verified that I can see "Invalid error code" works (https://github.com/altugbakan/rs-tftpd/blob/main/src/packet.rs#L277). 
But I am not seeing "File not found" error on the output.

Sample logs:
Here file test exists but abc and pqr doesn't. I expected "FileNotFound" to be logged but it isn't.

```
Sending test to <>
Invalid code error 
Sending test to <>
Sent test to <>
Sending abc to <>
Sending pqr to <>
```

**Comment by altugbakan** [comment]
Would you like to work on it?

**Comment by stappersg** [comment]
On Tue, Nov 05, 2024 at 10:54:24PM -0800, Altuğ Bakan wrote:
> 
> Would you like to work on it?
> 


There is a YES in [#32](https://github.com/altugbakan/rs-tftpd/pull/32)

**Comment by altugbakan** [comment]
Fixed with #32
